src/api/altorenderer.cpp

```diff
@@ -51,6 +51,20 @@ static void AddBoxToAlto(const ResultIterator *it, PageIteratorLevel level,
   }
 }
 
+static std::string GetID(char const * prefix, int page_number, int counter) {
```

Comment:
```suggestion
static std::string GetID(const char *prefix, int page_number, int counter) {
```

Reply:
The original code is technically fine, but my suggested change uses the coding style which is typical for Tesseract code.